### ISSUE TITLE
Cannot add Newznab indexer

### ISSUE THREAD

**Issue by nzbtuxnews** [Radarr/Radarr#2277]
I cannot add my local newznab indexer to Radarr.  I followed the same config as my running Sonarr setup to add NN+ to Radarr but for some reasons Radarr wont connect to it to test and will not allow me to save it.

I have entered the indexer's config like this in Radarr (same in Sonarr but with Sonarr it works flawlessly):

Name:  Local NN+ indexer
Enable RSS Sync = YES
Enable Search=YES
URL: https://newznabindexer/
API Key: some random string
Categories: 2000,2010,2020,2030,2035,2040,2045,2050,2060
Anime categories: blank
Additional parameters: blank
Remove year from search string: No
Search by Title: No

When I click the test button it says "Unable to connect to indexer, check the log for more details"

When I click "Save" the window flashes quickly and nothing happens.

Radarr  0.2.0.870

Radarr.txt:

[v0.2.0.870] System.InvalidOperationException: Sequence contains no elements
  at System.Linq.Enumerable.First[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00010] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 
  at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].TestConnection () [0x0001a] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 

17-11-19 11:30:09.1|Warn|NzbDroneErrorPipeline|Invalid request Validation failed: 
 -- Unable to connect to indexer, check the log for more details
17-11-19 11:31:17.0|Warn|Newznab|Unable to connect to indexer

[v0.2.0.870] System.InvalidOperationException: Sequence contains no elements
  at System.Linq.Enumerable.First[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00010] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 
  at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].TestConnection () [0x0001a] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 

17-11-19 11:31:17.0|Warn|NzbDroneErrorPipeline|Invalid request Validation failed: 
 -- Unable to connect to indexer, check the log for more details
17-11-19 11:31:18.9|Warn|Newznab|Unable to connect to indexer

[v0.2.0.870] System.InvalidOperationException: Sequence contains no elements
  at System.Linq.Enumerable.First[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00010] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 
  at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].TestConnection () [0x0001a] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 

17-11-19 11:31:18.9|Warn|NzbDroneErrorPipeline|Invalid request Validation failed: 
 -- Unable to connect to indexer, check the log for more details
17-11-19 11:31:19.0|Warn|Newznab|Unable to connect to indexer

[v0.2.0.870] System.InvalidOperationException: Sequence contains no elements
  at System.Linq.Enumerable.First[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00010] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 
  at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].TestConnection () [0x0001a] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 

17-11-19 11:31:19.0|Warn|NzbDroneErrorPipeline|Invalid request Validation failed: 
 -- Unable to connect to indexer, check the log for more details
17-11-19 11:31:19.2|Warn|Newznab|Unable to connect to indexer

[v0.2.0.870] System.InvalidOperationException: Sequence contains no elements
  at System.Linq.Enumerable.First[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00010] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 
  at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].TestConnection () [0x0001a] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 

17-11-19 11:31:19.2|Warn|NzbDroneErrorPipeline|Invalid request Validation failed: 
 -- Unable to connect to indexer, check the log for more details
17-11-19 11:31:19.3|Warn|Newznab|Unable to connect to indexer

[v0.2.0.870] System.InvalidOperationException: Sequence contains no elements
  at System.Linq.Enumerable.First[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00010] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 
  at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].TestConnection () [0x0001a] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 

17-11-19 11:31:19.3|Warn|NzbDroneErrorPipeline|Invalid request Validation failed: 
 -- Unable to connect to indexer, check the log for more details
17-11-19 11:31:19.5|Warn|Newznab|Unable to connect to indexer

[v0.2.0.870] System.InvalidOperationException: Sequence contains no elements
  at System.Linq.Enumerable.First[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x00010] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 
  at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].TestConnection () [0x0001a] in <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX>:0 

17-11-19 11:31:19.5|Warn|NzbDroneErrorPipeline|Invalid request Validation failed: 
 -- Unable to connect to indexer, check the log for more details
17-11-19 11:31:21.3|Warn|Newznab|Unable to connect to indexer

**Comment by tusharbhutt** [comment]
Site is down...


http://www.isitdownrightnow.com/newznabindexer.com.html

**Comment by nzbtuxnews** [comment]
I am NOT using an external NN indexer but a local (LAN) one which works.

**Comment by hotio** [comment]
Are you connecting to that local lan indexer over https? Is not going to fly.

**Comment by hotio** [comment]
movies vs tvshows, lots is different. You are not giving a whole lot of info to work with... "a local indexer"

**Comment by nzbtuxnews** [comment]
OK I am not sure what you want to know about my NN setup, but please ask and I will gladly provide.

So NN+ is a local indexer accessible at https://newznabindexer/

It was installed via SVN on a centos 7 VPS.  My media apps (Sonarr, Headphones, COuchpotato) are using my "user" account using my API key to access NN+ & download jobs.  The categories numbers I provided above (2000,2010, etc) are retrieved from the categories in NN+ and are matching for all media apps.

Other than that, I am not sure what else to provide.  Please ask.

**Comment by galli-leo** [comment]
This error is present when your indexer does not support searching by imdbid. I would suggest asking the developer to enable that. You can also add the indexer with Searching disabled and then reenable searching. Radarr will then just search by title instead of imdbid, which will be less reliable.

**Comment by nzbtuxnews** [comment]
OK my indexer supportas imdbid as per https://newznab.readthedocs.io/en/latest/misc/api/#movie-search

It is a vanilla NN install, straight from their SVN repo.  Nevertheless here's what I did:

Disabled both "Enable search" and "Enable RSS Sync".  Still could not test (connect to NN) but could save the config.

Went back to the config, and enabled "Search by title".  Still could not test (connect to NN) but could save the config.

I tried all kinds of configurations, to no avail.
